src/configure-nilrt-snac

```diff
@@ -140,6 +140,18 @@ remove_niauth() {
 	trap - EXIT
 }
 
+disable_wifi() {
+	log INFO Disabling WiFi...
+	cat <<EOT >/etc/modprobe.d/snac_blacklist.conf
+# Do not allow WiFi connections
+# We cannot use the blacklist keyword because they will still be loaded
+# when another module depends on them. This will prevent the modules from
+# being loaded along with any modules that depends on them.
+install cfg80211 /bin/true
```

Comment:
Is the return value of the right-hand side relevant? Initially I was thinking as a C developer "oh true means 1 which is a non-zero exit code so it prevents further processing" but then of course I tried it and realized the `true` command returns 0.

So if the return value is relevant would `/bin/false` which *does* return 1 be more correct? Or would that cause error messages to start spewing?

Reply:
/bin/true is the command we tell modprobe to run to "install" cfg80211. We want it to think it worked but not actually do anything, then the dependent modules will fail to load because their dependency is not loaded. Using /bin/false would indicate a failure to load, which is not what we want.